### PR TITLE
Fetch Cart: minicart initialisation

### PR DESCRIPTION
**Pull request by garmashdev**
# Description
Please include a summary of the change and which issue is fixed. Include the motivation for the changes, and comment on your PR if necessary for clarity.

Fixes: (link ticket)

#changelog Fetch Cart: minicart initialisation

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# For PR Reviewer 

- [ ] Reviewed unit tests to make sure we are using real components rather than mocks as much as possible?
- [ ] For any major change (observer, new Bolt feature, core Magento interaction) we must add a feature switch, did you verify this?

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my ticket link and provided a changelog message.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203268043628389

### CODE REVIEW

**Comment by garmashdev** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1660** <a href="https://app.graphite.dev/github/pr/BoltApp/bolt-magento2/1660" target="_blank">Graphite</a> 
    * **PR #1672** <a href="https://app.graphite.dev/github/pr/BoltApp/bolt-magento2/1672" target="_blank">Graphite</a> 
      * **PR #1667** <a href="https://app.graphite.dev/github/pr/BoltApp/bolt-magento2/1667" target="_blank">Graphite</a> 
        * **PR #1671** <a href="https://app.graphite.dev/github/pr/BoltApp/bolt-magento2/1671" target="_blank">Graphite</a> 
          * **PR #1681** <a href="https://app.graphite.dev/github/pr/BoltApp/bolt-magento2/1681" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/BoltApp/bolt-magento2/1681?utm_source=stack-comment).

File: view/frontend/web/js/bolt-api-driven-checkout.js [hunk 1]
```diff
@@ -36,56 +38,216 @@ define([
         magentoCartTimeStamp: null,
         cartBarrier: null,
         hintsBarrier: null,
+        elementBarrier: null,
```
Comment (vitaliyreznikov) [inline view/frontend/web/js/bolt-api-driven-checkout.js:41]: What does this name mean?

File: view/frontend/web/js/bolt-api-driven-checkout.js [hunk 2]
```diff
@@ -156,6 +320,239 @@ define([
...
+                     * On Magento checkout page - Bolt payment only checkout
+                     * trigger click on boltpay radio if the button clicked is
+                     * in minicart panel or other (voluntary) location
+                     * and trigger email validation.
+                     */
+                    if (BoltCheckoutApiDriven.getCheckoutType() === 'payment') {
+                        // trigger click on boltpay radio
+                        if ($('#boltpay').prop('checked') === false) $('#boltpay').click();
+
+                        // stop if customer email field exists and is not valid on paymtnt only page
```
Comment (vitaliyreznikov) [inline view/frontend/web/js/bolt-api-driven-checkout.js:506]: type: payment

File: view/frontend/web/js/bolt-api-driven-checkout.js (hunk 1 above)
Comment (garmashdev) [reply view/frontend/web/js/bolt-api-driven-checkout.js]: Updated to "ReadyStatus", prev. value in replacejs.phtml it was "Check" (like: checkPromise, resolveCheckPromise, etc.)

File: view/frontend/web/js/bolt-api-driven-checkout.js (hunk 2 above)
Comment (garmashdev) [reply view/frontend/web/js/bolt-api-driven-checkout.js]: Fixed.

**Review by vitaliyreznikov — APPROVED** [review]

**Pull request merged**